app.yaml

```diff
@@ -0,0 +1,18 @@
+name: starling
+
+image:
+  repository: quay.io/reevoo/starling
+
+pipeline:
+  tests:
+    spec:
+      script: rake spec
+      workdir: /usr/src
```

Comment:
Why do we need this here if the Dockerfile already has the WORKDIR set to `/usr/src` ?

Reply:
This is the dir that Jenkins runs in... it defaults to app...